control/xferfcn.py

```diff
@@ -1616,10 +1620,7 @@ def tf(*args, **kwargs):
 
     See Also
     --------
-    TransferFunction
-    ss
-    ss2tf
-    tf2ss
+    TransferFunction, ss, ss2tf, tf2ss
```

Comment:
`nlsys `

Reply:
I looked into this, and I don't think we need to reference `nlsys` here.  It is another factory function, but not that directly relevant to transfer functions.  I will include `nlsys` in the `ss` factory function docstring.